scroll-animations-1/Overview.bs

```diff
@@ -369,76 +372,81 @@ spec: cssom-view-1; type: dfn;
-				declared with the same name.
-
-			If more than one such candidate exists,
-			then this timeline has no [=attachment=].
+			This [=scroll progress timeline=] name’s [=attachment=]
+			is deferred to its descendants.
+			In other words,
+			this element declares a [=scroll progress timeline=]’s name
+			and defines its scope (see [[#timeline-scope]]),
+			but defers the timeline’s definition
+			([=scroll container=] source and axis)
+			to its descendants.
+
+			If more than one matching descendant [=scroll progress timeline=] exists,
+			or if no such timeline exists,
+			then the timeline name has no [=attachment=],
+			and represents an [=inactive timeline=].
+
+			ISSUE: Should it instead be ignored (be invisible to lookup)?
+			(This might make it harder to implement.)
+			If multiple, should it attach instead to the last declared timeline (in tree order)
+			rather than invalidating the attachment?
 
 		<dt><dfn>ancestor</dfn>
 		<dd>
-			The timeline has no [=attachment=].
-
-			Timelines with [=ancestor=] attachment are treated as nameless
-			for the purposes of timeline lookup;
-			they can't be referenced by 'animation-timeline'.
+			This [=scroll progress timeline=] definition,
+			attaches to the closest matching [=scroll progress timeline=] name
+			that was deferred (''scroll-timeline-name/defer'')
+			by an ancestor (if any).
+
+			If no such ancestor exists,
+			the [=attachment=] is treated as ''scroll-timeline-attachment/local''.
+
+			If this box is not a [=scroll container=], however,
+			then this [=scroll progress timeline=] definition is ignored.
+			<!-- Not only will it not bind to a name,
+			     it also won't invalidate any other bindings to that name. -->
 	</dl>
 
-	Timelines that have no [=attachment=] are [=inactive timeline|inactive=].
-
-	<div class=example>
-		Using ''scroll-timeline-attachment: defer'',
-		an element can effectively refer to timelines
-		that appear <em>after it</em> in tree order.
-		For example, the following creates an animation on <code>#target</code> that
-		is ultimately linked to a scroll timeline defined on the subsequent sibling.
-
-		<pre class=lang-css>
-		@keyframes anim {
-		  from { color: red; }
-		  to { color: green; }
-		}
-
-		#root {
-		  scroll-timeline-name: mytimeline;
-		  scroll-timeline-attachment: defer;
-		}
-
-		#root #target {
-		  animation: anim;
-		  animation-timeline: mytimeline;
-		}
-
-		#root #target + #scroller {
-		  scroll-timeline-name: mytimeline;
-		  scroll-timeline-attachment: ancestor;
-		}
-		</pre>
-	</div>
-
 ### Scroll Timeline Shorthand: the 'scroll-timeline' shorthand ### {#scroll-timeline-shorthand}
 
 	<pre class='propdef shorthand'>
 	Name: scroll-timeline
-	Value: [ <<'scroll-timeline-name'>> [ <<'scroll-timeline-axis'>> || <<'scroll-timeline-attachment'>> ]? ]#
+	Value: [ <<'scroll-timeline-name'>> [ <<'scroll-timeline-axis'>> | <<'scroll-timeline-attachment'>> ]? ]#
```

Comment:
As pointed out by flackr at some point, using `|` here means this won't be possible: 

`scroll-timeline: foo inline ancestor`.